api-code-examples/rust/docs-keys.rs

```diff
@@ -0,0 +1,39 @@
+import iroh
+import asyncio
+
+
+async def main():
```

Comment:
This one, too

Reply:
(and more)

Reply:
see the draft state? working my way through these to convert